templates/deployment.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Release.Name }}
+    spec:
+      containers:
+      - name: {{ .Release.Name }}
+        image: ghcr.io/glueops/qr-code-generator:latest:{{ .Chart.AppVersion }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+        - name: LOKI_API_ENDPOINT
+          value: {{ .Values.image.lokiApiEndpoint }}
+        ports:
+        - containerPort: 80
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .Release.Name }}
+spec:
+  selector:
+    app: {{ .Release.Name }}
+  ports:
+    - protocol: TCP
+      port: 80000
```

Comment:
80000 is invalid. I think the highest port number you could use is around 65k. ref: https://www.pico.net/kb/what-is-the-highest-tcp-port-number-allowed/#:~:text=The%20highest%20TCP%20port%20number%20is%2065%2C535.,16%20%2D%201)%20or%2065%2C535.

Reply:
an oversight, my bad